src/Resources/config/http_kernel.xml

```diff
@@ -13,5 +13,19 @@
             <argument/>
             <argument type="service" id="logger" on-invalid="null"/>
         </service>
+
+        <!-- Overrides Symfony's security.exception_listener -->
+        <service id="security.exception_listener" class="Sonata\PageBundle\Listener\SecurityExceptionListener" abstract="true">
```

Comment:
Having real qualms around this: this won't solve the issue if the user overrides the listener in their project for another reason. There must be a better way to do this... Maybe just creating an additional listener that would run after the symfony one?

Reply:
Possibly. But like you said in the other issue, let's see if there are better solutions for the problem on sonata level. I'll take this fix back to our project and just leave this here for reference for now.

Reply:
This is not acceptable I'm afraid. We can force an override of a Symfony service.